dask/dataframe/tests/test_ufunc.py

```diff
@@ -166,8 +166,6 @@ def test_ufunc_array_wrap(ufunc):
     - np.ufunc(dd.Series) => np.ndarray
     - np.ufunc(pd.Series) => np.ndarray
     """
-    if ufunc == "fix" and np.__version__ >= "1.13.0":
-        pytest.skip("fix calls floor in a way that we do not yet support")
```

Comment:
Hmm this seems to be causing errors still. Maybe we should just drop the `and np.__version__ >= "1.13.0"` part for now?

Reply:
Oh I didn't mean include that change :facepalm: